src/c_vec.rs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2020 Apple Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//! Slice C FFI
+
+extern crate libc;
```

Comment:
External crates should only ever be declared in the root module of the library, and added to the Cargo manifest so they're pulled in as a dependency when necessary.